lib/features/sign_up/view_model/sign_up_bloc.dart

```diff
@@ -65,12 +65,11 @@ class SignUpBloc extends Bloc<SignUpEvent, SignUpState> {
     if (displayName.isEmpty) {
       displayName = HatSpaceStrings.current
           .defaultUserDisplayName(DateTime.now().toUtc());
-      await _authenticationService.updateUserDisplayName(displayName);
```

Comment:
this change is here to only update user's display name if user does not have a display name. Please do not change this.